textx/metamodel.py

```diff
@@ -176,9 +178,16 @@ def __init__(self, file_name=None, classes=None, builtins=None,
 
         # Convert classes to dict for easier lookup
         self.user_classes = {}
+
+        def default_user_classes_provider(name):
+            return None
+        self.user_classes_provider = default_user_classes_provider
```

Comment:
Here I suggest just setting `self.user_classes_provider` to `None`.

Reply:
ok